### ISSUE TITLE
Create a template for the example gallery

### ISSUE THREAD

**Issue by moorepants** [pydy/pydy.github.io.src#8]
This should extend `page.html` and provide a grid structure for the example thumbnails. We need to figure out how to make Sphinx use this new template only for the example gallery page.

**Comment by moorepants** [comment]
Here is a hack method of having specific templates for specific pages:
http://stackoverflow.com/questions/13209597/override-html-page-template-for-a-specific-sphinx-document

**Comment by moorepants** [comment]
@jcrist The above is the best thing I can find. I'm surprised I can't find anything else about this on the web. It seems like it would be a common feature request for Sphinx since all generic static site generators support ways to deploy different templates to different pages.

**Comment by jcrist** [comment]
Yeah, it doesn't seem easily doable. That solution is kinda hackish, but not _terrible_. If we're only doing it for one page it shouldn't be too bad.

**Comment by moorepants** [comment]
Yeah, I think this is the best bet. I looked pretty hard for other options. I'll get it setup this weekend.

Can you check into loading the correct css and stuff into sphinx to support notebook formats?

We will also need a template for "notebook" pages.

**Comment by moorepants** [comment]
May be helpful: http://sphinx-ipynb.readthedocs.org/en/latest/howto.html

**Comment by moorepants** [comment]
We now have an examples gallery in the pydy documentation. It could be nice to somehow have a master examples gallery for sympy + pydy + opty, etc. But closing for now.
